authors/apps/articles/models.py

```diff
@@ -1,47 +0,0 @@
-from django.db import models
```

Comment:
Why are you deleting everything in this file?

Reply:
I got rid of the articles folder and intergrated the changes to article folder. Thus rendering this file unwanted.